From 7c84a6ae8cab29a8eb1270b8c1de0cb346210df2 Mon Sep 17 00:00:00 2001
From: Loic Devulder <ldevulder@suse.com>
Date: Fri, 17 Apr 2020 17:02:39 +0200
Subject: [PATCH] Fix EC2 stonith plugin

The 'profile' parameter is not handled correctly in EC2 stonith plugin.
This commit fix it using the same way as the 'tag' variable.

Origin: upstream, https://github.com/ClusterLabs/cluster-glue/commit/7c84a6ae8cab29a8eb1270b8c1de0cb346210df2
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1895355
Reviewed-by: Rafael David Tinoco <rafaeldtinoco@ubuntu.com>
Last-Update: 2020-09-15
---
 lib/plugins/stonith/external/ec2 | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

--- a/lib/plugins/stonith/external/ec2
+++ b/lib/plugins/stonith/external/ec2
@@ -294,12 +294,7 @@
 [ -n "$1" ] && action=$1
 [ -n "$2" ] && node_to_fence=$2
 
-if [ -z "$ec2_profile"]; then
-	options="--output text --profile default"
-else
-	options="--output text --profile $ec2_profile "
-fi
-
+options="--output text --profile $ec2_profile"
 action=`echo $action | tr 'A-Z' 'a-z'`
 
 case $action in
